frontend/integration-tests-cypress/cypress/tests/monitoring_spec.ts

```diff
@@ -0,0 +1,97 @@
+import { testName, submitButton, errorMessage } from '../support/index';
+import { timeouts } from '../support/utils';
+
+const testDetailsPage = (pageTitle: string, sectionHeading: string, testLabel = true) => {
+  cy.getTestElement('resource-title', timeouts.pageLoad).contains(pageTitle); // long timeout for detail page to load
+  cy.getTestElement('section-heading')
+    .first()
+    .should('have.text', sectionHeading);
+  if (testLabel) {
+    cy.get('.co-m-label')
+      .first()
+      .contains(`alertname=${pageTitle}`);
+  }
+};
+
+describe('Monitoring: Alerts', () => {
+  // TODO: need to move these to before/after all specs run,
+  //  not just before/after this spec
+  before(() => {
+    cy.createTestProject(testName);
+  });
+  after(() => {
+    cy.deleteTestProject(testName);
+  });
+
+  it('displays the Alerts list page', () => {
+    cy.clickNavLink(['Monitoring', 'Alerting']);
+    cy.getTestElement('resource-title').should('have.text', 'Alert');
+  });
+
+  it('does not have a namespace dropdown', () => {
+    cy.getTestElement('namespace-bar-dropdown').should('not.exist');
+  });
+
+  it('filters Alerts by name', () => {
+    cy.filterListPageByName('Watchdog');
+    cy.getTestElement('alert-resource-link')
+      .should('have.length', 1)
+      .should('have.text', 'Watchdog');
+  });
+
+  it('displays Alert details page', () => {
+    cy.getTestElement('alert-resource-link')
+      .first()
+      .should('have.text', 'Watchdog')
+      .click();
+    testDetailsPage('Watchdog', 'Alert Details');
+  });
+
+  it('links to the Alerting Rule details page', () => {
+    cy.getTestElement('alert-rules-detail-resource-link')
+      .contains('Watchdog')
+      .click();
+    testDetailsPage('Watchdog', 'Alerting Rule Details', false);
```

Comment:
Inclined to make this more descriptive.  I'd be inclinedto make this an object w/funcs:

```typescript
testDetailsPage
   .hasTitle('Watchdog')
   .hasSectionHeading('Alerting Rule Details')
   .hasTitleLabel(false)
```
or just simply

```typescript 
pagehasTitle('Watchdog');
pageHasHeading('Alerting Rule Details')
pageHasTitleLabel('Watchdog')
```
That may be more verbose than you want, but I think the clearer we make the tests the better in the long run.

Reply:
Updated to use calls like:
```
    listPage.filter.byName('Watchdog'); // needed 'pipe' for more specificity due to fuzzy search
    listPage.rows.countShouldBe(1);
    listPage.rows.shouldExist('Watchdog');
```
and
```
  detailsPage.titleShouldBe('Watchdog');
  detailsPage.sectionHeaderShouldExist('Silence Details');
  detailsPage.labelShouldExist('alertname=Watchdog');
```
These are now located at `frontend/integration-tests-cypress/cypress/support/views/details-page.ts & list-page.ts`